.github/workflows/check-pl-compat.yaml

```diff
@@ -152,7 +152,7 @@ jobs:
         echo "PYTHONPATH=$PYTHONPATH:$(pwd)/quantum-build/python_packages/quantum" >> $GITHUB_ENV
         echo "RUNTIME_LIB_DIR=$(pwd)/runtime-build/lib" >> $GITHUB_ENV
         echo "MLIR_LIB_DIR=$(pwd)/llvm-build/lib" >> $GITHUB_ENV
-        echo "ENZYME_DIR=$(pwd)/enzyme-build/Enzyme" >> $GITHUB_ENV
+        echo "ENZYME_LIB_DIR=$(pwd)/enzyme-build/Enzyme" >> $GITHUB_ENV
```

Comment:
Would you like to run CPL checks to ensure there's nothing else needed with those actions?

Reply:
They don't build wheels so I think we're good.

Reply:
I meant checking the build with regards to the Enzyme lib path updates. :sweat_smile: